sdcm/coredump.py

```diff
@@ -202,12 +202,13 @@ def extract_info_from_core_pids(
     def _upload_coredump(self, core_info: CoreDumpInfo):
         coredump = core_info.corefile
         coredump = self._pack_coredump(coredump)
-        base_upload_url = 'upload.scylladb.com/%s/%s'
+        base_upload_url = 'https://upload.scylladb.com/%s/%s'
         coredump_id = os.path.basename(coredump)[:-3]
         upload_url = base_upload_url % (coredump_id, os.path.basename(coredump))
         self.log.info('Uploading coredump %s to %s' % (coredump, upload_url))
-        self.node.remoter.run("sudo curl --request PUT --upload-file "
+        self.node.remoter.run("sudo curl --request PUT --fail --show-error --upload-file "
                               "'%s' '%s'" % (coredump, upload_url))
+        upload_url = upload_url.replace('https://', '')
```

Comment:
The need to have such a line signals that the change is not as good as it can be.
On the line 214 we see the reason - usage of the different protocol than the `https` - `gs`.
So, the proper way, I think, would be to do the same on the line 209 - place `https` protocol directly in the command like on line 214 and keep the `base_upload_url` as is. In such a case you won't need to have such strange lines like this one - 211.

Reply:
fixed